mathcomp/algebra/dune

```diff
@@ -0,0 +1,19 @@
+(coq.theory
+ (name mathcomp.algebra)
+ (package coq-mathcomp-algebra)
+ (modules :standard)
+ (synopsis "Mathematical Components")
```

Comment:
The metadata is sketchy, the one in the opam files is much better. Example:
```
synopsis: "Mathematical Components Library on Algebra"
description: """
This library contains definitions and theorems about discrete
(i.e. with decidable equality) algebraic structures : ring, fields,
ordered fields, real fields,  modules, algebras, integers, rational
numbers, polynomials, matrices, vector spaces...
"""
```

Reply:
I guess this metadata is used in order to generate the opam packages, but I may be wrong.

Reply:
Oh indeed I didn't go the effort to update the metadata until you folks discussed if this would be accepted. Will fix.

Reply:
Ok, meta-data update, noted that I didn't enable the opam file generation yet as it would overwrite the opam files in the repos.

Reply:
Actually the meta-data here is for the META file, which is not relevant here. If we have something similar to META files for Coq one day it would become relevant.